src/LogicTasks/Syntax/ComposeFormula.hs

```diff
@@ -79,7 +79,7 @@ description path ComposeFormulaInst{..} = do
     basicOpKey unicodeAllowed
     arrowsKey
 
-    paragraph $ indent $ do
+    when inputHelp $ paragraph $ indent $ do
```

Comment:
Vielleicht ist hier das vollständige Unterdrücken der Hilfe zu rigide. Die angezeigte Information bezieht sich ja nicht nur auf die Listenform, sondern auch darauf, wie die einzelnen Formeln in der Liste aussehen. Also vielleicht sollte es stattdessen so sein, dass wenn das Flag auf True gesetzt ist, das gleiche wie zuvor angezeigt wird (mit der Form `[(A ∨ ¬B) and C, C and (A or not B)]`) und ansonsten etwas in der Art angezeigt wird. "Ein Lösungsversuch könnte beispielsweise aus den folgenden beiden Eingaben bestehen: `(A ∨ ¬B) and C`, `C and (A or not B)`."

Reply:
Oder es wird halt an anderer Stelle der andere Text eingefügt, wie hier vorgeschlagen (https://github.com/fmidue/flex-tasks/issues/2#issuecomment-2429488471):
> Das kann dann aber entweder in Flex-Konfigurationen